test/inductor/test_compiled_optimizers.py

```diff
@@ -68,14 +68,16 @@ class KernelCounts(NamedTuple):
     RMSprop: KernelCounts(multitensor=1, singletensor=4),
     Adadelta: KernelCounts(multitensor=1, singletensor=4),
     Adagrad: KernelCounts(multitensor=5, singletensor=8),
-    ASGD: KernelCounts(multitensor=2, singletensor=12),
+    ASGD: KernelCounts(
+        multitensor=2, singletensor=None
```

Comment:
discussed offline, we shouldn't disable this test, just handle the capturable check by gating on if we are compiling or not

Reply:
done